examples/multithread_test.rs

```diff
@@ -0,0 +1,84 @@
+// Regression test for https://github.com/psychon/x11rb/issues/231
+
+use std::sync::Arc;
+
+use x11rb::connection::Connection as _;
+use x11rb::generated::xproto::{
+    ClientMessageData, ClientMessageEvent, ConnectionExt as _, CreateWindowAux, EventMask,
+    WindowClass, CLIENT_MESSAGE_EVENT,
+};
+use x11rb::x11_utils::Event as _;
+use x11rb::COPY_DEPTH_FROM_PARENT;
+
+fn main() {
+    let (conn, screen_num) = x11rb::connect(None).unwrap();
+    let conn = Arc::new(conn);
+
+    let screen = &conn.setup().roots[screen_num];
+    let window = conn.generate_id();
+    conn.create_window(
+        COPY_DEPTH_FROM_PARENT,
+        window,
+        screen.root,
+        0, // x
+        0, // y
+        1, // width
+        1, // height
+        0,
+        WindowClass::InputOutput,
+        screen.root_visual,
+        &CreateWindowAux::new().event_mask(EventMask::NoEvent),
```

Comment:
Isn't that the default? In other words, is this equivalent to just `&CreateWindowAux::new()`? (or `::default()` or `Default::default()`)

Reply:
Indeed, it can left just as `CreateWindowAux::new()`.

Reply:
...but it also does not matter much